R/radial_bridge.R

```diff
@@ -365,3 +381,33 @@ change_coordinates <- function(obj, ...) {
   obj$alpha <- alpha[seq(from = 1, to = length(alpha), by = 5)]
 
 }
+
+
+#Model evaluation
+#Requires: brain class type.
+qmodel.brain<- function(data, type="wildtype", threshold.n=0.9){
+  if (type == "wildtype"){
+    tidy_brain <- data %>%
+      tidy(type="wildtype", threshold = threshold.n )
+    ro_tidy_brain<- reorient(tidy_brain)
+  }
+  if(type== "youtoo"){
+    tidy_brain <- data%>%
+      tidy(type="youtoo", threshold = threshold.n)
+    ro_tidy_brain<- reorient(tidy_brain)
+  }
```

Comment:
Can't we just pass through the `type`? So it would just be 
```
ro_tidy_brain <- data %>%
    tidy(type, threshold = threshold.n ) %>%
    reorient()
```